README.md

```diff
@@ -8,3 +8,51 @@ https://kudacam.netlify.app/
 
 Maintainer 
 [Vighnesh Manjrekar](https://github.com/VighneshManjrekar)
+
+# Hacktoberfest22
+## KickStart to Open Source Event 
+
+## What is Hacktoberfest?
+A month-long celebration from October 1st - 31st sponsored by [Digital Ocean](https://hacktoberfest.digitalocean.com/) and [GitHub](https://github.com/blog/2433-celebrate-open-source-this-october-with-hacktoberfest) to get people involved in [Open Source](https://github.com/open-source). Create your very first pull request and contribute to the open source developer community.
```

Comment:
Create your very first pull request and contribute to the open source developer community.
 Remove this line